docusaurus/docs/guides/deployments/10-linux/10-controller/10-deploy.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Controller Deployment
+sidebar_label: Controller
+id: deploy
+---
+
+import LinuxPackageRepo from '/docs/_linux-package-repo.mdx';
+import CliLogFormats from '/docs/_cli-log-levels-and-formats.md';
+
+This article is about deploying a Ziti Controller as a Linux system service. [The controller introduction](/learn/introduction/03-components.md#openziti-controller) introduces the architectural concepts used in this guide.
+
+## Installation
+
+The controller package `openziti-controller` provides a systemd service unit and optional bootstrapping. The package depends on the `openziti` package which provides the `ziti` CLI. The easiest way to install both is the cross-platform install script for RPM and Debian distributions.
+
+```text
+curl -sS https://get.openziti.io/install.bash \
+| sudo bash -s openziti-controller
+```
+
+<LinuxPackageRepo />
+
+## Bootstrapping
+
+You can opt-in to generating a PKI, config, and database by installing interactively with prompts or by setting these values.
+
+1. In `/opt/openziti/etc/controller/service.env` (the answer file for the service)
+    1. Set `ZITI_BOOTSTRAP=true`
+1. In `/opt/openziti/etc/controller/bootstrap.env` (the answer file for bootstrapping the controller)
+    1. Set `ZITI_CTRL_ADVERTISED_ADDRESS` to the FQDN of the controller
+    1. Set `ZITI_PWD` to the desired management password for user `admin`. You may delete this after bootstrapping.
+
+### Generating a Public Key Infrastructure
+
+The controller service will generate a root CA and issue an intermediate CA cert during the first startup.
+
+The controller needs a CA to issue certificates for edge identities during enrollment. This CA is known as the edge signer. It's a good idea to secure the root in a different location and use an intermediate CA for this purpose. This allows you to recover in a scenario where the intermediate was lost without changing the root of trust and invalidating all edge enrollments.
+
+Disable bootstrapping a PKI by setting `ZITI_BOOTSTRAP_PKI=false` in `/opt/openziti/etc/controller/service.env`.
+
+Check out [the PKI page](/learn/core-concepts/pki.md) for an overview of these concepts.
+
+### Generating a Configuration
+
+The controller service will generate a configuration during the first startup.
+
+The controller's configuration is loaded from a YAML file ([reference](/reference/30-configuration/controller.md)). The Linux system service will generate a valid configuration during the first startup unless one already exists. 

```

Comment:
Some, if not all of this, feels redundant to "The controller service will generate a configuration during the first startup." I also think it's not necessary to tell people it's loaded from a YAML file and just reference "the controller's configuration file".